### DISCUSSION TITLE
Locate BCM module in a righ hand drive Renault Clio

### DISCUSSION THREAD

**Discussion by dvdking14** [cedricp/ddt4all#846] [Q&A]
As the title suggests I am unable to locate the BCM module inside a right hand drive Renault Clio iconic (model year 2020) is there anyone who managed to access the BCM in the RHD clio? If so would it be possible to provide a guide to do so?

**Comment by KarelSvo** [comment]
Can't find the ECU in the vehicle to connect directly to it?
Or does the software you use not find the control unit?
Is the cabin fuse box located differently on LHD and RHD?

**Reply by dvdking14** [reply]
The Fuse box is located in the left hand side of the car. It takes part of the glovebox space. Usually the BCM module on a Left hand drive car is located behind the glovebox however I am not sure if it is located in the same location or not.

**Comment by matzel82** [comment]
Normally on the side where the steeringweel is, so behind the fuse-box or the OBD Connection. Du you have a VIN ? Than I can take a look at our Service Portal

**Reply by dvdking14** [reply]
Yes I have a vin number. Do you have an email I can send it to you? My car is a 2020 Clio Iconic  Right hand drive model.

**Comment by eldimoni1** [comment]
Aquí te mando una fotografía con volante a la izquierda 
![Uploading image.jpeg…]()
